tests/utils.py

```diff
@@ -168,3 +169,34 @@ def assertTensorEqual(self, *args, **kwargs):  # noqa: N802
 
     def assertTensorAlmostEqual(self, *args, **kwargs):  # noqa: N802
         assert_array_almost_equal(*args, **kwargs)
+
+    def get_h5DS_path(
+            self,
+            stem,
+            binary=False,
+            shape=(10, 20, 30),
+            spacing=(1, 1, 1),
+            components=1,
+            add_nans=False,
+            force_binary_foreground=True,
+            no_channel_dim=False
+            ):
+        shape = (*shape, 1) if len(shape) == 2 else shape
+        if no_channel_dim:
+            data = np.random.rand(*shape)
+        else:
+            data = np.random.rand(components, *shape)
+        if binary:
+            data = (data > 0.5).astype(np.uint8)
+            if not data.sum() and force_binary_foreground:
+                data[..., 0] = 1
```

Comment:
Can we cover this line in the tests?

Reply:
Already done. There is a test for it "test_label_map_type_h5ds"